kubernetes/samples/scripts/create-weblogic-domain/model-in-image/model-configmaps/workmanager/model.20.workmanager.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+resources:
+  SelfTuning:
+    MinThreadsConstraint:
+      SampleMinThreads:
+        Count: 2
+    MaxThreadsConstraint:
+      SampleMaxThreads:
+        Count: 20
+  JDBCSystemResource:
```

Comment:
I thought the JDBC SR was going to be removed from here?  (Since the original yaml for the JDBC SR is now loaded into the new configmap too...)

Reply:
Sorry, missed this file in previous commit. Should be fixed now. Thanks!